cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -305,6 +314,16 @@ public boolean isExportedEquipment(Identifiable<?> c) {
         return !ignored;
     }
 
+    public CgmesExportContext setBoundaryEqId(String boundaryEqId) {
```

Comment:
Is it possible to store BD_EQ and BD_TP ids during import (as properties) and automatically filled the context during export? It can be done in another PR (if you prefer to do it later, I will create an issue)

Reply:
Yes, for networks coming from CGMES we could use the approach you suggest. It is ok for me to have a separate PR